__tests__/app.test.js

```diff
@@ -50,4 +50,25 @@ describe('/api/articles', () => {
             })
         })
     });
+});
+
+describe('/api/articles/:article_id/comments', () => {
+    test('GET request responds with an array of comments for the given article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(( { body }) => {
+            console.log(body)
+            expect(body.length).toBeGreaterThan(0)
```

Comment:
The body itself should not be the array of comments - it should be an object with the key of `comments` - on that key is the array